common/lib/xmodule/xmodule/modulestore/search.py

```diff
@@ -3,12 +3,12 @@
 
 from logging import getLogger
 
-
```

Comment:
since the changes in the file amount to noise at this point, could you restore the original file so that it's not part of this diff?

you should be able to do this for any file with `git checkout origin/master -- <filename>`.

please do it with all the `common/static` and `common/test` folders as well -- it looks like your editor added newline or two to those.